gulp/favicons.js

```diff
@@ -0,0 +1,115 @@
+const { Readable } = require('stream');
+const Vinyl = require('vinyl');
+const gulp = require('gulp');
+const favicons = require('favicons');
+const jsonEditor = require('gulp-json-editor');
+const rename = require('gulp-rename');
+const filter = require('gulp-filter');
+const gravityParticles = require('@buildit/gravity-particles');
+
+const optimiseImages = require('./image-optim-pipe');
+const gulpConfig = require('../config/gulp.json');
+const siteConfig = require('../config/site.json').site;
+
+// eslint-disable-next-line prefer-destructuring
+const paths = gulpConfig.paths;
```

Comment:
If we prefer destructuring then surely we should destructure here? `const { paths } = gulpConfig;`